user_tools/src/spark_rapids_pytools/resources/templates/dataproc-create_gpu_cluster_script.ms

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+export CLUSTER_NAME="{{ CLUSTER_NAME }}"
+export REGION="{{ REGION }}"
+export ZONE="{{ ZONE }}"
```

Comment:
is ZONE needed?  i get that REGION is used twice so you can keep that there

Reply:
Thanks @mattahrens !
I removed the extra exports and added the bootstrap command